### ISSUE TITLE
Converting Units to Account For New Data Types

### ISSUE THREAD

**Issue by victor-vazquez** [Phlya/adjustText#16]
I just stumbled across this package. Good stuff! I see an opportunity for improvement though. 

Right now this package will not correctly handle datatypes that matplotlib supports, for example Pandas Period datatypes. The fault lies in your use of `text.get_position()`. That return raw data values, not converted data values. If you replace:

```python
x, y = text.get_position()
```

with 

```python
def get_text_position(text, ax=None):
    ax = ax or plt.gca()
    x, y = text.get_position()
    return (ax.xaxis.convert_units(x),
            ax.yaxis.convert_units(y))

x, y = get_text_position(text)
```

it will support all data types that matplotlib currently supports.

**Comment by Phlya** [comment]
Thanks!

Interesting... Have you tried this? Can you make a PR?

**Comment by victor-vazquez** [comment]
Yeah. I've made the code change locally and tested it. I tried pushing a branch to this repo, but I don't think I have permissions.

**Comment by Phlya** [comment]
Thanks for clarifying!

I didn't suggest you to push into the repository, since indeed you don't have the permission, but you can create a pull request, which I can then merge into my branch of the code: https://help.github.com/articles/about-pull-requests/ This would be the best way to include your code into the package since this would allow to keep track of who introduced it. Please try to do this, but if you have problems I can include this into my code directly too, of course.
